dataplane/saiserver/hostif.go

```diff
@@ -170,6 +178,22 @@ func (hostif *hostif) CreateHostif(ctx context.Context, req *saipb.CreateHostifR
 				},
 			},
 		}
+
+		// Unless, the corresponding port for this hostif is the CPU port, then run the normal forwarding pipeline.
+		if resp.GetAttr().GetCpuPort() == req.GetObjId() {
+			update = &fwdpb.PortUpdateRequest{
+				ContextId: &fwdpb.ContextId{Id: hostif.dataplane.ID()},
+				PortId:    &fwdpb.PortId{ObjectId: &fwdpb.ObjectId{Id: fmt.Sprint(id)}},
+				Update: &fwdpb.PortUpdateDesc{
+					Port: &fwdpb.PortUpdateDesc_Kernel{
+						Kernel: &fwdpb.KernelPortUpdateDesc{
+							Inputs: getForwardingPipeline(),
```

Comment:
Inputs field is the only field changed compared to the update above. To avoid dup code, can we only change this field?

Reply:
done